src/rest/Facet/FacetSearchResponse.ts

```diff
@@ -16,6 +17,10 @@ export interface IFacetSearchResultValue {
    * the filter expression that would get generated if the facet value were selected.
    */
   count: number;
+  /**
+   * Whether additional facet values matching the request are available.
+   */
+  moreValuesAvailable: boolean;
```

Comment:
This description is copied from the [Swagger documentation for Search API](https://platformdev.cloud.coveo.com/docs/?api=SearchApi#!/Search/post_rest_search_v2_facet).

Reply:
I don't think it's used, but maybe it's still work documenting here. As one day a customer could request loading more search results somehow which is not a feature we have or planned for yet.

I would remove it from the rest of the code for now, since it's not used and can lead to confusion (in IDynamicHierarchicalFacetSearchValueProperties, DynamicHierarchicalFacetSearch.ts, DynamicHierarchicalFacetSearchValue.ts).

If (big if) we add the "show more search results feature" in the future, we'd better do it for both Hierarchical & Specific type facets.

Reply:
Good point, I'll remove it.